### PR TITLE
improve rounding on analytics, add path to scroll percentage

### PR DESCRIPTION
**Pull request by robert-j-webb**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -310,26 +310,26 @@ $(document).ready(function () {
...
     if (!timeStartedOnPage) {
       return;
     }
-    const durationInSeconds = (new Date().getTime() - timeStartedOnPage.getTime()) / 1000;
+    const durationInSeconds = Math.round(new Date().getTime() - timeStartedOnPage.getTime()) / 1000;
     amplitude.track('timeOnPage', {
       duration: `${Math.round(durationInSeconds)}`,
       minutes: `${Math.round(durationInSeconds / 60)}`,
-      path,
+      pathname,
```
Comment (serenackuo) [inline src/index.js:321]: There are a few different property names for path in Amplitude (path, pathname, location). It seems like location has the most number of events associated. Can we make this property `location` for consistency? We can then use a property transform in Amplitude to make the rest consistent if we want.

File: src/index.js [hunk 2]
```diff
@@ -310,26 +310,26 @@ $(document).ready(function () {
...
   let timeStartedOnPage = new Date();
 
-  function trackTimeOnPage(path) {
+  function trackTimeOnPage(pathname) {
     if (!timeStartedOnPage) {
       return;
     }
-    const durationInSeconds = (new Date().getTime() - timeStartedOnPage.getTime()) / 1000;
+    const durationInSeconds = Math.round(new Date().getTime() - timeStartedOnPage.getTime()) / 1000;
     amplitude.track('timeOnPage', {
```
Comment (serenackuo) [inline src/index.js:318]: This should autoformat, but can we change it to `TimeOnPage` to match the event naming convention for Amplitude?

File: src/index.js (hunk 1 above)
Comment (robert-j-webb) [reply src/index.js:321]: No, location is something else. Location is place within the page, path name is the URL of the page

File: src/index.js (hunk 2 above)
Comment (robert-j-webb) [reply src/index.js:318]: Yeah I believe amplitude auto fixes this stuff

**Review by serenackuo — APPROVED** [review]

**Pull request merged**